### PR TITLE
Jira demo fixes

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Jira Xsiam Demo fixes

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by richardbluestone — APPROVED** [review]

File: Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.yml [hunk 1]
```diff
@@ -32,19 +32,7 @@ configuration:
...
-  display: Vendor name
-  name: vendor
-  required: false
-  type: 0
-- defaultvalue: jira
-  additionalinfo: "For the dataset name for example: jira"
-  display: Product name
-  name: product
-  required: false
-  type: 0
```
Comment (yaakovi) [inline Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.yml:46]: Why did you remove those?

**Review by yaakovi — APPROVED** [review]

**Comment by yaakovi** [comment]
validation failed on parameter removal
